Week 05/Yoana Borisova SR

```diff
@@ -0,0 +1,97 @@
+void Swap(int& num1, int& num2)
+{
+    int temp = num1;
+    num1 = num2;
+    num2 = temp;
+}
+
+int* getArr(int num)
+{
+    const size_t ARRAY_SIZE = 10;
+    int arr[ARRAY_SIZE];
+
+    arr[0] = num;
+
+    return arr; 
+}
+
+int main()
+{
+    const size_t ARRAY_SIZE = 1024;
+    int arr1[ARRAY_SIZE];
+    size_t array1Size;    
+    std::cout << "Enter number of elements: ";
+    std::cin >> array1Size;
+    if (!std::cin)
+    {
+        std::cerr << "Could not read number of elements.";
+    }
+    if (array1Size > ARRAY_SIZE)
+    {
+        std::cerr << "Invalid number of elements. Max number of elements is " << ARRAY_SIZE;
+    }
+
+    int arr2[ARRAY_SIZE];
+    size_t array2Size;
+    std::cout << "Enter number of elements: ";
+    std::cin >> array2Size;
+    if (!std::cin)
+    {
+        std::cerr << "Could not read number of elements.";
+    }
+    if (array2Size > ARRAY_SIZE)
+    {
+        std::cerr << "Invalid number of elements. Max number of elements is " << ARRAY_SIZE;
+    }
+
+    //first array
+    std::cout << "Enter the elements of first array: ";
+    for (size_t i = 0; i < array1Size; i++)
+    {
+        std::cin >> arr1[i];
+    }
+    if (std::cin.bad())
+    {
+        std::cerr << "Could not read elements of array.";
+    }
+    //second array
+    std::cout << "Enter the elements of second array: ";
+    for (size_t i = 0; i < array2Size; i++)
+    {
+        std::cin >> arr2[i];
+    }
+    if (std::cin.bad())
+    {
+        std::cerr << "Could not read elements of array.";
+    }
+
+    int array3Size = array1Size + array2Size;
+    int arr3[array3Size];
```

Comment:
You should not create static arrays using a variable. Use predefined constant intead.